datasets/doqa/doqa.py

```
@@ -0,0 +1,184 @@
+                    "text": nlp.Value("string"),
+                    "answer_start": nlp.Value("int32"),
+            }),
+                "followup": nlp.Value("string"),
+                "yesno": nlp.Value("string"),
+                "orig_answer": nlp.features.Sequence({
+                    "text": nlp.Value("string"),
+                    "answer_start": nlp.Value("int32"),
+            }),
+            }),
+            homepage="http://ixa.eus/node/12931",
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        path = dl_manager.download_and_extract(_URL)
+        if self.config.name == 'cooking':
+            return [
+                nlp.SplitGenerator(
+                    name=nlp.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset",  "doqa-cooking-test-v2.1.json")},
+                ),
+                nlp.SplitGenerator(
+                    name=nlp.Split.VALIDATION,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset", "doqa-cooking-dev-v2.1.json")},
+                ),
+                nlp.SplitGenerator(
+                    name=nlp.Split.TRAIN,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1","doqa_dataset",  "doqa-cooking-train-v2.1.json")},
+                )
+            ]
+        elif self.config.name == "movies":
+            return [
+                nlp.SplitGenerator(
+                    name=nlp.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset",  "doqa-movies-test-v2.1.json")},
+                )
+            ]
+        elif self.config.name == "travel":
+            return [
+                nlp.SplitGenerator(
+                    name=nlp.Split.TEST,
+                    gen_kwargs={"filepath": os.path.join(path, "doqa-v2.1", "doqa_dataset", "doqa-travel-test-v2.1.json")},
+                )
+            ]
```

maybe raise an error if the config name is wrong ?